src/components/MultiAssetPreview/MultiAssetPreview.spec.tsx

```diff
@@ -0,0 +1,57 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import React from "react";
+import { MultiAssetPreview } from "./MultiAssetPreview";
+
+const MULTI_ASSET_PREVIEW_ID = "[data-test-id=multi-asset-preview]";
+
+describe("MultiAssetPreview Component", () => {
+    it("should render without images", () => {
+        mount(<MultiAssetPreview onClick={() => alert("Multi Asset clicked")} />);
+
+        cy.get(MULTI_ASSET_PREVIEW_ID).should("exist");
+    });
+
+    it("should render with images", () => {
+        mount(
+            <MultiAssetPreview
+                onClick={() => alert("Multi Asset clicked")}
+                numberOfLocations={12}
+                assetsAmount={25}
+                images={[
```

Comment:
To make the test a bit more clear, I'd suggest you create an `EXAMPLE_IMAGES` variable and pass it in here